CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+Substrate Sidecar API is an Open Source project that welcomes new issues and PRs.
+
+If you’re new to open source or interested in contributing, this guide: [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/) is particularly helpful.
+
+## Contributing Code
+To contribute to this repo, it’s recommended to create an issue first; this allows you to gather feedback before writing any code. In the issues tab, you’ll be able to select from a template to create an issue. 
+
+To create a Pull Request after you’ve created an issue, you’ll need to head over to the Pull Request tab. This will be where all the PRs will be located, and where reviewers will be able to review your changes made. 
+
+## Rules
+There are a few basic ground-rules for contributors (including the maintainer(s) of the project):
+- No `--force` pushes or modifying the Git history in any way. If you need to rebase, ensure you do it in your own repo.
```

Comment:
I personally do not agree with this rule. I think we can use GitHub to protect branches that shouldn't be force-pushed, but I think it is fine for developers to fore-push to their own branches off this repo.

Reply:
Hm, I see what you mean there. Sometimes a force push is necessary, even though it may not always be recommended. I'll modify that sentence to something like "Try to avoid..."